CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Below are the most important changes from each release.
 A more detailed list of changes is available in the corresponding milestones for each release in the Github issue tracker (https://github.com/googlefonts/fontbakery/milestones?state=closed).
 
+## TBD
+### Miscellaneous
```

Comment:
please always add these entries under the upcoming release session (0.6.13 this time)